src/confcom/azext_confcom/security_policy.py

```diff
@@ -446,15 +446,35 @@ def populate_policy_content_for_all_images(
                 if (
                     isinstance(image, UserContainerImage) or individual_image
                 ) and image_info:
-                    # verify and populate the startup command
-                    if not image.get_command():
+                    # verify and populate the startup command for VN2 since "command" and "args"
+                    # can be set independent of each other. These names correspond to what we call
+                    # "entrypoint" and "command"
+                    # entrypoint should be None for everything except VN2
+                    image_entrypoint = image.get_entrypoint()
+                    if image_entrypoint is not None:
+                        image_command = image.get_command()
+                        manifest_entrypoint = image_info.get("Entrypoint") or []
+                        manifest_command = image_info.get("Cmd") or []
+                        # this describes the potential options that can happen: https://unofficial-kubernetes.readthedocs.io/en/latest/concepts/configuration/container-command-args/
+                        if image_entrypoint and not image_command:
+                            command = image_entrypoint
+                        elif image_entrypoint and image_command:
+                            command = image_entrypoint + image_command
+                        elif image_command:
+                            command = manifest_entrypoint + image_command
+                        else:
+                            command = manifest_entrypoint + manifest_command
+                        image.set_command(command)
+
+                    elif not image.get_command():
```

Comment:
Can this be an else? Or is there some other option?

Reply:
The "other" option is covered when the image object is created. That's when the defaults are used, this is overwriting the defaults when those arguments are supplied by the user